CITATION.cff

```diff
@@ -0,0 +1,45 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+authors:
+  - given-names: Tangi
+    family-names: Migot
+    email: tangi.migot@gmail.com
+    affiliation: >-
+      GERAD and Department of Mathematics and
+      Industrial Engineering, Polytechnique Montréal,
+      QC, Canada.
+    orcid: 'https://orcid.org/0000-0001-7729-2513'
+  - affiliation: >-
+      GERAD and Department of Mathematics and
+      Industrial Engineering, Polytechnique Montréal,
+      QC, Canada
+    family-names: Orban
+    given-names: Dominique
+    orcid: 'https://orcid.org/0000-0002-8017-7687'
+    email: dominique.orban@gerad.ca
+  - affiliation: 'Netherlands eScience Center, Amsterdam, NL'
+    family-names: Soares Siqueira
+    given-names: Abel
+    orcid: 'https://orcid.org/0000-0003-4451-281X'
+    email: abel.s.siqueira@gmail.com
+  - family-names: contributors
```

Comment:
Could you please put the fields in the same order for each author?

Reply:
Oops sorry for that, that was pretty random